### PR TITLE
Allow user custom fzf-tags's prompt

### PR DESCRIPTION
**Pull request by luathn**
This plugin is awesome.
So I want it to be more configurable. This is the first one.
Hope you feel comfortable with it!
```vim
let g:fzf_tags_prompt = "Gd "
```
2020-12-25_133247-810x475_scrot

### CODE REVIEW

File: autoload/fzf_tags.vim [hunk 1]
```diff
@@ -32,10 +32,13 @@ function! fzf_tags#Find(identifier)
     execute 'tag' identifier
   else
     let expect_keys = join(keys(s:actions), ',')
+    if !exists('g:fzf_tags_prompt')
```
Comment (zackhsi) [inline autoload/fzf_tags.vim]: As you're planning to add more customizations, this makes more sense at the top of the file, just before `let s:actions...`.

**Review by zackhsi — CHANGES_REQUESTED** [review]
Thanks for the contribution! I've let one small nit, but other than that, it looks great 👍

File: autoload/fzf_tags.vim (hunk 1 above)
Comment (luathn) [reply autoload/fzf_tags.vim]: @zackhsi It's good idea. Thank you.
I fixed it. Please check it out!

**Review by zackhsi — APPROVED** [review]
Thanks 👍

**Pull request merged**